source/web/css/guidelines.css

```diff
@@ -0,0 +1,22 @@
+/* layout for MEI Guidelines */
+p, div.p {}
```

Comment:
empty rule?

Reply:
This CSS is a bare bone file to be populated later. This line here is a placeholder indicating that the other rules are not overwritten for now.

Reply:
I see your point, but my CSS analysis tool would throw me a "major code smell" for empty blocks: https://rules.sonarsource.com/css/RSPEC-4658

So maybe remove it now and include it as soon as it is really used. Or, if it needs to be kept, I would suggest to add a comment inside the empty declaration that this is intentionally left empty until implementation. Otherwise, my future self will always stumble upon this empty rule and bug you with questions about why it's there ;)

Reply:
removed `guidelines.css`